File: anagram.py
def are_anagrams(a,b):
	word1 ="".join(sorted(list(a.lower())))
	word2 ="".join(sorted(list(b.lower())))
	if (word1 == word2):
		print "words are anagram"
	else:
		print "words are not anagram"
	

File: anagram.js
function are_anagrams(a,b){
	var word1 = a.replace(/[^A-Za-z]/, '').toLowerCase().split("").sort().join("");
	var word2 = b.replace(/[^A-Za-z]/, '').toLowerCase().split("").sort().join("");
	if (word1 == word2){
		console.log("words are anagrams");
	} else {
		console.log("words are not anagrams");
	}
}
are_anagrams(a,b);